target/linux/lantiq/xrx200/base-files/etc/hotplug.d/firmware/12-ath9k-eeprom

```diff
@@ -34,7 +34,6 @@ case "$FIRMWARE" in
 				;;
 			tplink,tdw8970|\
 			tplink,tdw8980)
```

Comment:
This case statement is not needed any more.

Reply:
If I remove the case statement, wouldn't it print "caldata: board $board is not supported yet" to the bootlog? It would confuse people.

Reply:
ath9k should not request the blob from userspace anymore since it can retrieve it using in-kernel code (that means: no need to call userspace anymore).
since userspace (12-ath9k-eeprom script) is not called anymore that warning should not be printed, even if you remove this case statement

that said, this is how it's supposed to work - I haven't actually tested it

Reply:
Yeah, it looks like it doesn't call the userspace blob if it retrieves it's firmware via nvmem.